### ISSUE TITLE
`gamepad`: Clarify what storing settings in the project means

### ISSUE THREAD

**Issue by DNin01** [ScratchAddons/ScratchAddons#5458]
### Why this would be helpful

There are a few things the "Store these settings in the project to override the default configuration (Experimental tool for project creators)" checkbox could mean. Maybe it's to have SA remember these settings. Or maybe to save the settings somewhere inside the project (this is the answer).

"Experimental tool for project creators" does give us a hint, but it's still generally vague.

Even then, this switch is useless for anyone but the project creator (or people downloading the project to play it in TurboWarp or remixing it). Also, the creator has to enter the editor and save the project for the configurations to apply to others, I'm assuming.

And for anyone else, they won't be the owner of the project in order to save their edits (unless they make a remix), so they can't save the settings, but the way it's worded makes it sound like it might get saved in SA, if you're not reading carefully. These users may see it **and might misunderstand and think that SA would remember the controls they configured** when it actually doesn't - this actually has to be inserted into the project by its creator.

### My suggestion

We could do one of a few things:

1. Reword it to better describe the function. Perhaps "Store these settings **inside** the project"?
2. Add an extended description like "the creator of a project can use this setting to store the controls above inside the project so that other Scratch Addons users playing it will have the same controls".
3. Only make it visible to the project creator.
4. If you aren't the creator, have SA remember the configuration too.

**Comment by penguinmoose** [comment]
I like 1 and 3. It doesn't do anything if you're not the owner of the project.

If we have SA remember the configuration for the project if you don't own it (4), the description needs to be changed accordingly.

I don't think an extended description is necessary. The reworded description (1) is pretty self-explanatory, although if we want to emphasize that other users will also have the same settings, we can add "so other people have the same settings". A sentence that long is too much.
